### PR TITLE
Update Experiment table to MUI

### PR DESCRIPTION
**Pull request by pbadillatorrealba**
This pr updates the table of experiments to the MUI components.

The changes are: 
- The experiment table and new experiment button is now implemented using MUI's Datagrid and Button.
- A title and a container (Paper) are added to the experiment table.
- The component is completely defined in its file (before it was between main.jsx and experimentstable.jsx).

Note: this pr depends of the approvation of #35

### CODE REVIEW

File: DashAI/front/src/components/ExperimentsTable.jsx [hunk 1]
```diff
@@ -1,73 +1,131 @@
...
+  const [rows, setRows] = React.useState(initialRows);
+
+  const deleteUser = React.useCallback(
+    (id) => () => {
+      setTimeout(() => {
+        setRows((prevRows) => prevRows.filter((row) => row.id !== id));
+      });
+    },
+    []
+  );
```
Comment (cristian-tamblay) [inline DashAI/front/src/components/ExperimentsTable.jsx]:
```suggestion
  const deleteExperiment = React.useCallback(
    (id) => () => {
      setTimeout(() => {
        setRows((prevRows) => prevRows.filter((row) => row.id !== id));
      });
    },
    []
  );
```
I guess endpoint call should be added here as WIP and not only shadow delete?

File: DashAI/front/src/components/ExperimentsTable.jsx [hunk 2]
```diff
@@ -1,73 +1,131 @@
...
+          <GridActionsCellItem
+            key="delete-button"
+            icon={<DeleteIcon />}
+            label="Delete"
+            onClick={deleteUser(params.id)}
+          />,
+        ],
+      },
+    ],
+    [deleteUser]
```
Comment (cristian-tamblay) [inline DashAI/front/src/components/ExperimentsTable.jsx]:
```suggestion
    [deleteExperiment]
```

File: DashAI/front/src/components/ExperimentsTable.jsx [hunk 3]
```diff
@@ -1,73 +1,131 @@
...
+  const formatDate = (date) => {
+    if (date == null) {
+      return "";
+    }
+
+    const formattedDate =
+      date.getDate() + "/" + (date.getMonth() + 1) + "/" + date.getFullYear();
+
+    return formattedDate;
+  };
```
Comment (cristian-tamblay) [inline DashAI/front/src/components/ExperimentsTable.jsx]: This function does not depends on component, maybe move it to `src/utils`?

File: DashAI/front/src/components/ExperimentsTable.jsx [hunk 4]
```diff
@@ -1,73 +1,131 @@
...
+      {
+        field: "actions",
+        type: "actions",
+        minWidth: 80,
+        getActions: (params) => [
+          <GridActionsCellItem
+            key="delete-button"
+            icon={<DeleteIcon />}
+            label="Delete"
+            onClick={deleteUser(params.id)}
```
Comment (cristian-tamblay) [inline DashAI/front/src/components/ExperimentsTable.jsx]:
```suggestion
            onClick={deleteExperiment(params.id)}
```

**Review by cristian-tamblay — CHANGES_REQUESTED** [review]

File: DashAI/front/src/components/ExperimentsTable.jsx (hunk 1 above)
Comment (pbadillatorrealba) [reply DashAI/front/src/components/ExperimentsTable.jsx]: I forgot to change the name, it came this way from the template. 
I will change the name to make the merge correct.
Anyway, the next pr should extract the code from the component requests. In particular, this request should be in `api/experiment.ts`.

File: DashAI/front/src/components/ExperimentsTable.jsx (hunk 3 above)
Comment (pbadillatorrealba) [reply DashAI/front/src/components/ExperimentsTable.jsx]: OK, I externalized it to `utils/index` and added the import at the beginning of the file.

File: DashAI/front/src/components/ExperimentsTable.jsx (hunk 4 above)
Comment (pbadillatorrealba) [reply DashAI/front/src/components/ExperimentsTable.jsx]: Ok

**Review by cristian-tamblay — APPROVED** [review]

**Pull request merged**